srcpkgs/e16/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'e16'
+pkgname=e16
+version=1.0.22
+revision=1
+archs="i686 x86_64"
```

Comment:
Remove. This seems to build at least on `armv7l` and there's no reason it shouldn't build on all architectures.